From b05af104d7b52956304476f3c613892d2d0f53ae Mon Sep 17 00:00:00 2001
From: Richard Muzik <richard.muzik@turris.com>
Date: Mon, 29 Sep 2025 08:51:07 +0200
Subject: [PATCH] syslog-ng: Add missing dependency on libunwind

If libunwind is compiled before syslog-ng the build fails with missing
dependency error.

Fixes:

```
Package syslog-ng is missing dependencies for the following libraries:
libunwind.so.8
```

Signed-off-by: Richard Muzik <richard.muzik@turris.com>
---
 admin/syslog-ng/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/admin/syslog-ng/Makefile b/admin/syslog-ng/Makefile
index 36de97a59..54bfc5ab1 100644
--- a/admin/syslog-ng/Makefile
+++ b/admin/syslog-ng/Makefile
@@ -34,7 +34,7 @@ define Package/syslog-ng
   CATEGORY:=Administration
   TITLE:=A powerful syslog daemon
   URL:=https://www.syslog-ng.com/products/open-source-log-management/
-  DEPENDS:=+libpcre2 +glib2 +libopenssl +libpthread +librt +libdbi +libjson-c +libcurl +libuuid +SYSLOGNG_LOGROTATE:logrotate +LIBCURL_ZLIB:zlib
+  DEPENDS:=+libpcre2 +glib2 +libopenssl +libpthread +librt +libdbi +libjson-c +libcurl +libuuid +SYSLOGNG_LOGROTATE:logrotate +LIBCURL_ZLIB:zlib +libunwind
   ALTERNATIVES:=300:/sbin/logread:/usr/libexec/logread.sh
 endef
 
-- 
2.51.0

